client/go/vespa/id.go

```diff
@@ -0,0 +1,46 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+// vespa document ids
+// Author: bratseth
+
+package vespa
+
+import (
+	"errors"
+	"strings"
+)
+
+func IdToUrlPath(documentId string) (string, error) {
```

Comment:
Nit: The Go convention is to use normal capitalization for abbreviations (e.g. `func IDToURLPath(documentID string)`), unlike the Java convention.